src/TodoApp.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TodoApp = ({ createNewTodo, addNewTodo, enterNewTodo }) => (
```

Comment:
```suggestion
const TodoInput = ({ createNewTodo, addNewTodo, enterNewTodo }) => (
```
Это у тебя компонент ввода, а не все приложение